mnist/README.md

```diff
@@ -181,23 +110,14 @@ The argo workflow can be daunting, but basically our steps above extrapolate as
 
 With our workflow defined, we can now execute it.
 
-## Submitting your training workflow
-
-First we need to set a few variables in our workflow. Make sure to set your docker registry or remove the `IMAGE` parameters in order to use our defaults:
-
 ```
-DOCKER_BASE_URL=docker.io/elsonrodriguez # Put your docker registry here
 export S3_DATA_URL=s3://${BUCKET_NAME}/data/mnist/
 export S3_TRAIN_BASE_URL=s3://${BUCKET_NAME}/models
-export JOB_NAME=myjob-$(uuidgen  | cut -c -5 | tr '[:upper:]' '[:lower:]')
-export TF_MODEL_IMAGE=${DOCKER_BASE_URL}/mytfmodel:1.7
+export JOB_NAME=myjob1
+export TF_MODEL_IMAGE=siji/mnist-model:v1.11.0
```

Comment:
How and where is this model container big built? Can we host it in kubeflow-examples?
You can push images to that repo from our CI system
https://github.com/kubeflow/kubeflow/blob/master/docs_dev/releasing.md#image-auto-release

Reply:
I prefer to change this one by one since I don't want make PR depends on each other. after this PR get merged, I can submit new PR to fix the image issue.

Reply:
Can you open an issue please to track this and reference it in a TODO?

Reply:
@jlewi The image was being built by the reader in the previous version, the instructions on how to build/push were deleted in this update.

Reply:
The example images will be published in kubeflow which will be addressed in this separate issue: https://github.com/kubeflow/examples/issues/319

The instructions may not work well in future, the packages changes a lot, the only way to make it work all the time is pre-build it and publish to kubeflow.

Users just need to get it and use it, most users will not build the images by themself, if they want they can build them, later I will update the details in issue: https://github.com/kubeflow/examples/issues/319

This PR is too large, I want to reduce the changes and make it merged fast.